tools/cldr-apps/src/test/java/org/unicode/cldr/web/TestVettingViewerQueue.java

```diff
@@ -0,0 +1,75 @@
+package org.unicode.cldr.web;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.SQLException;
+
+import org.json.JSONException;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.unicode.cldr.unittest.web.TestAll;
+import org.unicode.cldr.util.Organization;
+import org.unicode.cldr.util.VoteResolver;
+import org.unicode.cldr.web.VettingViewerQueue.LoadingPolicy;
+
+public class TestVettingViewerQueue {
+    /**
+     * Test the Vetting Viewer Summary
+     */
+    @Disabled("VERY SLOW")
+    @Test
+    void testSummaryResponse() throws IOException, JSONException, SQLException, InterruptedException {
```

Comment:
it will be good to have a test

maybe VettingViewer and VettingViewerQueue could have options used only for unit testing, causing them to use smaller sets of locales and paths so they'll be fast enough